bitnami/grafana-loki/values.yaml

```diff
@@ -200,9 +200,10 @@ loki:
             max_size_items: 1024
             validity: 24h
           {{- end }}
-
+    {{- if not .Values.queryScheduler.enabled }}
```

Comment:
This is affecting to all grafana-loki componente, not only the querier. Is it expected?